curricula/templates/curricula/partials/lesson_pills.html

```diff
@@ -3,7 +3,10 @@
 <div class="resource-nav btn-toolbar" role="toolbar" aria-label="Lesson Resources">
     <div class="btn-group btn-group-xs" role="group" aria-label="PDF Buttons">
         <a href="{{ unit.get_absolute_url }}" class="btn" role="button">{% trans 'Unit Overview' %}</a>
+        {# Removing lessons PDF link from CSF until we can fix it #}
+        {% if unit.has_resource_pdf %}
```

Comment:
Worth noting that this is likely to cause us tickets in A-F where those PDFs do work. Is it worth hard coding this to just target express/pre-express until we can fix?